src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+public class Todo extends Task {
+
+    /**
+     * Todo constructor
+     *
+     * @param description
+     * @param done
+     */
+    public Todo(String description, boolean done) {
+        super(description, done);
```

Comment:
I think maybe renaming the boolean to isDone, instead of done, will make it more obvious that done is a boolean.